lib/android/app/src/main/java/com/reactnativenavigation/views/BottomTabs.java

```diff
@@ -17,9 +18,10 @@ public BottomTabs(Context context) {
     }
 
     public void setTabTag(int index, Text testId) {
-        if (!testId.hasValue()) return;
-        if (testId.hasValue()) getViewAtPosition(index).setTag(testId.get());
-        if (testId.hasValue()) getViewAtPosition(index).setContentDescription(testId.get());
+        View view = getViewAtPosition(index);
+        if (!testId.hasValue() || view == null) return;
```

Comment:
I'm still not sold on this null check. If the tab has a `testId`, we should crash if the view is null, otherwise our e2e will be flaky.